include/clients/rt/LoudnessClient.hpp

```diff
@@ -33,6 +34,7 @@ enum LoudnessParamIndex {
 };
 
 constexpr auto LoudnessParams = defineParameters(
+    ChoicesParam("select","Selection of Outputs","loudness","truepeak"),
```

Comment:
this isn't always true peak, so should probably just say peak

Reply:
Indeed - will change

Reply:
Resolved in 3baac1d